lib/PuppeteerSharp.Tests/Frame/EvaluateTests.cs

```diff
@@ -17,8 +17,8 @@ public class EvaluateTests : PuppeteerPageBaseTest
             var frame1 = Page.Frames.ElementAt(0);
             var frame2 = Page.Frames.ElementAt(1);
 
-            await frame1.EvaluateExpressionAsync("window.FOO = 'foo'");
-            await frame2.EvaluateExpressionAsync("window.FOO = 'bar'");
+            await frame1.EvaluateExpressionHandleAsync("window.FOO = 'foo'");
+            await frame2.EvaluateExpressionHandleAsync("window.FOO = 'bar'");
```

Comment:
One thing I noticed across this PR (I won't mention this many times only here). Is that we're switching from `EvaluateExpression` to `EvaluateExpressionHandle` and that's not equivalent because the first one is disposing the handle as soon as possible.

Reply:
Just a thought. How about if we get rid of `EvaluateExpressionHandleAsync` and we treat `EvaluateExpressionAsync<JSHandle>` in a special way?

Reply:
puppeteer has both a `evaluate` and `evaluateHandle` functions, and I prefer not to add **magic** to the function

Reply:
@Meir017  👆 the first comment